src/risedevtool/src/task/configure_tmux_service.rs

```diff
@@ -54,47 +55,23 @@ impl Task for ConfigureTmuxTask {
             )
         })?;
 
-        // List previous windows and kill them
-        let mut cmd = self.tmux();
-        cmd.arg("list-windows")
-            .arg("-t")
-            .arg(RISEDEV_SESSION_NAME)
-            .arg("-F")
-            .arg("#{pane_id} #{pane_pid} #{window_name}");
-
-        if let Ok(output) = ctx.run_command(cmd) {
-            for line in String::from_utf8(output.stdout)?.split('\n') {
-                if line.trim().is_empty() {
-                    continue;
-                }
-                let [pane_id, pid, name, ..] = line.split(' ').collect_vec()[..] else {
-                    return Err(anyhow!("failed to parse tmux list-windows output"));
-                };
-                ctx.pb
-                    .set_message(format!("killing {} {} {}...", pane_id, pid, name));
-
-                // Send ^C & ^D
-                let mut cmd = self.tmux();
-                cmd.arg("send-keys")
-                    .arg("-t")
-                    .arg(pane_id)
-                    .arg("C-c")
-                    .arg("C-d");
-                ctx.run_command(cmd)?;
-            }
-        }
+        let mut cmd = new_tmux_command();
+        cmd.arg("list-sessions");
+        if ctx.run_command(cmd).is_ok() {
+            ctx.pb.set_message("killing previous session...");
 
-        let mut cmd = self.tmux();
-        cmd.arg("kill-session").arg("-t").arg(RISEDEV_SESSION_NAME);
-        ctx.run_command(cmd).ok();
+            let mut cmd = Command::new("./risedev");
```

Comment:
I would suggest not relying on `cargo make` here. 😕

Reply:
Then we have to duplicate the `wait_kafka_exit` and `wait_zookeeper_exit` here, otherwise kafka will fail to start, which is the case before this PR.

Reply:
Also, `minio` doesn't work even after we correctly killing and restart it, because it persists some ID. IMO it's more reasonable to notify the user, who is a dev, that "you have to run `risedev k` before run `d` again".

Reply:
Yes, that's also my concern, so I think it's better that we just let the users kill the cluster by themselves.